### PR TITLE
Add manager alert support

### PR DESCRIPTION
**Pull request by athiruma**
* disabled the manager alerts for resources: empty_roles, s3_inactive, nat_gateway_unused, ip_unattached <br/>
### The following rules will be applied.
1. ec2_idle -  manager in alert
2. ec2_stop -  manager  in alert
3. ebs_unattached -  manager  in alert
4. zombie_cluster_resource - No Alerts
5. zombie_snapshots  -  manager in alert
6. zombie_elastic_ips  - NO Need manager in alert
7. zombie_nat_gateways - NO Need manager in alert
8. s3_inactive - NO Need manager in alert
9. empty_roles - NO Need manager in alert

### CODE REVIEW

File: jenkins/clouds/aws/daily/policies/run_policies.py [hunk 1]
```diff
@@ -65,18 +65,21 @@ def get_policies(type: str = None):
...
+        elif policy in ('ec2_idle', 'zombie_snapshots', 'ec2_stop', 'ebs_unattached'):
+            os.system(f"""podman run --rm --name cloud-governance --net="host" -e MANAGER_EMAIL_ALERT="True" -e account="PERF-DEPT" -e policy="{policy}" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF}" -e AWS_DEFAULT_REGION="{region}" -e dry_run="no" -e LDAP_HOST_NAME="{LDAP_HOST_NAME}" -e special_user_mails="{special_user_mails}" -e account_admin="{account_admin}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}" -e policy_output="s3://{BUCKET_PERF}/{LOGS}/{region}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
```
Comment (ebattat) [inline jenkins/clouds/aws/daily/policies/run_policies.py:69]: MANAGER_EMAIL_ALERT="True"
This variable should be True by default in environment variable no need to pass it
Same for [EMAIL_ALERT="True" ](https://github.com/redhat-performance/cloud-governance/blob/fba1f79be2cbf7ee0bec843150862f04fbd2308e/cloud_governance/main/environment_variables.py#L159)
Meaning that we pass those parameters to the container only when its False.
Can you please fix it for both variables ?

Comment (athiruma) [reply jenkins/clouds/aws/daily/policies/run_policies.py]: Done

**Review by ebattat — APPROVED** [review]
/LGTM

File: jenkins/clouds/aws/daily/policies/run_policies.py (hunk 1 above)
Comment (ebattat) [reply jenkins/clouds/aws/daily/policies/run_policies.py]: @athiruma, do you think we need to add those env variable to readme or do it in other PR ?

**Pull request merged**